tools/gn

```diff
@@ -165,7 +165,7 @@ def get_host_cpu():
   if is_rosetta():
     return 'arm64'
   machine = platform.machine()
-  if machine in ['aarch64', 'arm64']:
+  if machine in ['aarch64', 'arm64', 'ARM64']:
```

Comment:
Should we just lower-case `machine` instead of duplicating this with a different case?

Reply:
I thought about doing this, but found "explicit" case to be a clearer choice. I can still change that though.

Reply:
I also lean a bit towards explicit values (even if it's just as a signal to the reader that different OSes report different values). The Dart SDK lists them explicitly as well:
https://github.com/dart-lang/sdk/blob/68951ae5a1834826a961f81706bed2706cf731fc/tools/utils.py#L168-L188